### DISCUSSION TITLE
Resolving merge conflicts in package-lock.json

### DISCUSSION THREAD

**Discussion by nimonian** [swe-delivery/swe-tube#14] [Q&A] [answered]
#11 has merge conflicts for package-lock.json

I could probably do this manually, but it feels wrong.

Is there a better way?

I can well imagine this is the kind of thing which would hamstring apprentices for a few hours (why can't I add my code!) and would be good to have some steps for them to follow.

**Comment by accuser** [comment, marked as answer]
Typically happens when multiple branches are merged asynchronously. I'll pick up the PR as is, pull or rebase from develop and commit, which will resolve the merge conflict.
